openff/toolkit/topology/molecule.py

```diff
@@ -3495,11 +3501,11 @@ def to_hill_formula(self) -> str:
 
         Returns
         ----------
-        formula : the Hill formula of the molecule
+        formula
 
         Raises
         -----------
-        NotImplementedError : if the molecule is not of one of the specified types.
+        NotImplementedError
```

Comment:
yeah the `Returns` sections in general don't add much information, I'd be okay with removing them in general (especially since it often forces the creation of a variable that just wastes space instead of returning when the result is ready to go)

anyway, for this bit I've just removed everything after the first line; it's a method with no arguments and a return value that should be fairly obvious, so I think it's fine to leave like that